src/SFA.DAS.Roatp.ProviderModeration.Web/Infrastructure/RouteNames.cs

```diff
@@ -10,5 +10,8 @@ public static class RouteNames
         public const string GetProviderDetails = "provider-details";
         public const string GetAddProviderDescription = "add-provider-description";
         public const string PostAddProviderDescription = "add-provider-description";
+        public const string GetReviewProviderDescription = "review-provider-description";
+        public const string GetReviewProviderDescriptionEdit = "review-provider-description-edit";
```

Comment:
This might be a better url as 'edit-provider-description' instead of 'review-provider-description-edit'